### ISSUE TITLE
Add Entropic Retraining

### ISSUE THREAD

**Issue by philipsperl** [fra31/auto-attack#42]
**Paper**: [Optimizing Information Loss Towards Robust Neural Networks](https://arxiv.org/abs/2008.03072)

**Venue**: [DYNAMICS 2020 – DYnamic and Novel Advances in Machine Learning and Intelligent Cyber Security](https://www.acsac.org/2020/workshops/dynamics/)

**Dataset and threat model**: CIFAR-10, L2, eps = 0.5

**Code**: Not published

**Pre-trained model**: Pre-trained model available [here](https://owncloud.fraunhofer.de/index.php/s/05oQYKNbZ5Bm2q0)

**Log file**: Log file of the evaluation available [here](https://owncloud.fraunhofer.de/index.php/s/70oHgFBo2FIW14t)

**Additional data**: No

**Clean and robust accuracy**: Clean accuracy: 81.75%. Robust accuracy: 56.67%

**Architecture**: Simple convolutional network: Four convolutional layers, the first two with filter of size 32, the second pair with filter size of 64. Each pair of convolutional layers is followed by a max-pooling layer. Finally, one flatten layer and six dense layers with 512, 256, 128, 128, 84, and 10 neurons each.

**Description of the model/defense**: New loss: During training, which is based normal data only, an adapted loss function is used. The additional loss-term is based on information theoretic inspired metrics. The method does not require the generation of adversarial examples during training.
